cmake/cmake_uninstall.cmake.in

```diff
@@ -10,7 +10,7 @@ foreach(file ${files})
 	message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
 	if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
 		execute_process(
-			COMMAND @CMAKE_COMMAND@ -E remove $ENV{DESTDIR}${file}
+			COMMAND rm $ENV{DESTDIR}${file}
```

Comment:
`rm` is not a fully portable command.

Since `@CMAKE_COMMAND@ -E remove` is deprecated since version 3.17 (see https://cmake.org/cmake/help/latest/manual/cmake.1.html),
I would suggest to use `@CMAKE_COMMAND@ -E rm` (new from version 3.17)
or just `file(REMOVE [<files>...])` (https://cmake.org/cmake/help/latest/command/file.html)
(both of them work well - verified)

Reply:
I've tries the first one, same behavior as currently. Will try the second one, thanks.

Reply:
The second one works as expected, updated, thanks!